sdk/core/azure-core-http-netty/README.md

```diff
@@ -15,7 +15,7 @@ Azure Core Netty HTTP client is a plugin for the azure-core HTTP client API.
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-core-http-netty</artifactId>
-    <version>1.2.0</version>
+    <version>1.3.0-beta.1</version>
```

Comment:
Non-existent version

Reply:
@JonathanGiles @joshfree Do we want to be listing most recently released version or currently being worked on versions in our READMEs

Reply:
My personal preference is to have the readme's always point users to the most recently released version, rather than the next unreleased version that the user can't actually pull down from Maven yet. It means that they can copy/paste and get working, rather than copy/paste and wonder why Maven can't find the referenced version.

Reply:
+1